scripts/superdesk-authoring/authoring.js

```diff
@@ -384,6 +384,17 @@ import 'angular-history/history.js';
             });
         };
 
+        this.validateBeforeTansa = function (orig, diff) {
```

Comment:
This function, on its own, seems unrelated to tansa. Would it not be better suited to call it simply `validate` ? Correct me if I'm wrong.